src/services/lease-service/adapters/offer-adapter.ts

```diff
@@ -4,45 +4,175 @@ import {
   Applicant,
   DetailedOffer,
   OfferStatus,
+  ApplicantStatus,
+  LeaseStatus,
 } from 'onecore-types'
 import { logger } from 'onecore-utilities'
 import { Knex } from 'knex'
 
 import { db } from './db'
-import { AdapterResult, DbApplicant, DbDetailedOffer, DbOffer } from './types'
+import {
+  AdapterResult,
+  DbApplicant,
+  DbDetailedOffer,
+  DbOffer,
+  DbOfferApplicant,
+} from './types'
 
 import * as dbUtils from './utils'
 
-type CreateOfferParams = Omit<
-  Offer,
-  'id' | 'sentAt' | 'answeredAt' | 'offeredApplicant' | 'createdAt'
-> & { applicantId: number }
-
-export async function create(params: CreateOfferParams) {
-  const { selectedApplicants, ...rest } = params
-  const values = {
-    ...rest,
-    selectionSnapshot: JSON.stringify(selectedApplicants),
-  }
+export type OfferApplicant = {
```

Comment:
I'm sure you already keep track of this but this could be imported from types once merged, right?

Reply:
Done